src/components/ui/ControlBar/ControlBar.stories.tsx

```diff
@@ -82,7 +87,37 @@ export const ControlBarForDocs = () => {
 
   return (
     <ControlBar showLabels layout="left" css="position: absolute;">
-      <ControlBarItem {...microphoneButtonProps} />
+      <ControlBarItem {...microphoneButtonProps}>
+        <PopOverHeader title="Title text" subtitle="Subtitle text" />
+        <PopOverItem
+          as="button"
+          onClick={() => console.log('clicked')}
+          children={<span>Also test content</span>}
+        />
+        <PopOverSeparator />
+        <PopOverItem
+          as="button"
+          onClick={() => console.log('clicked')}
+          children={<span>This is more test content</span>}
+        />
+        <PopOverSubMenu text="This is a submenu">
+          <PopOverItem
+            as="button"
+            onClick={() => console.log('clicked')}
+            children={<span>This is also a submenu component</span>}
+          />
```

Comment:
Nit - I know this is just a story, but can we use the implicit form of children instead of explicit?

```
<PopoverItem>
  <span>This is a submenu</span>
</PopoverItem>
```

Reply:
Fair enough, will update.